cmd/activator/main.go

```diff
@@ -165,6 +165,7 @@ func main() {
 	// Enable TLS client when queue-proxy-ca is specified.
 	// At this moment activator with TLS does not disable HTTP.
 	// See also https://github.com/knative/serving/issues/12808.
+	// Also, the current activator must be restarted when updating the secret of CA.
```

Comment:
(Thinking out loud) It seems that a cert pool uses a map underneath would it be  possible to override the existing rootCA pool when [the secret is updated](https://stackoverflow.com/questions/70084865/update-certpool-at-server-runtime)? It seems golang libs dont support client certs dynamic loading.

Reply:
Thank you for the comment.
Please let me try to solve only server cert part in this PR. Even only server certs part, it is not easy as some CI is failing and need to figure out it.